specs/core/0_beacon-chain.md

```diff
@@ -575,13 +582,16 @@ def bytes_to_int(data: bytes) -> uint64:
 
 `def hash_tree_root(object: SSZSerializable) -> Root` is a function for hashing objects into a single root by utilizing a hash tree structure, as defined in the [SSZ spec](../simple-serialize.md#merkleization).
 
-#### `bls_verify`
+#### BLS Signatures
 
-`bls_verify` is a function for verifying a BLS signature, as defined in the [BLS Signature spec](../bls_signature.md#bls_verify).
+Eth2 makes use of BLS signatures as specified in the [IETF draft BLS specification](https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-00). Specifically, eth2 uses the `BLS_SIG_BLS12381G2-SHA256-SSWU-RO-_POP_` ciphersuite which implements the following interfaces:
 
-#### `bls_aggregate_pubkeys`
+* `def Sign(SK: int, message: Bytes) -> BLSSignature`
+* `def Verify(PK: BLSPubkey, message: Bytes, signature: BLSSignature) -> bool`
+* `def Aggregate(signatures: Sequence[BLSSignature]) -> BLSSignature`
```

Comment:
Do we need `Aggregate`?

Reply:
We use `Sign` and `Aggregate` in the validator doc. For simplicity in specifying the BLS methods in one place, I'm in favor of putting them here

Reply:
There's a similar thing going on with `BLS_WITHDRAWAL_PREFIX`.

Reply:
> There's a similar thing going on with BLS_WITHDRAWAL_PREFIX.

I didn't want to touch this due to it not strictly being related to the IETF BLS standards and because it needs to be fixed with either a different hack in the spec builder or change the way the PySpec SSZ works. The latter option was taken in #1551 .